src/EditorFeatures/TestUtilities/PullMemberUp/PullMemberUpCodeActionTest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities.PullMemberUp
+{
+    public abstract class PullMemberUpCodeActionTest : AbstractCodeActionTest
+    {
+        internal Task TestWithPullMemberDialogAsync(
+            string initialMarkUp,
+            string expectedResult,
+            IEnumerable<(string, bool)> selection = null,
+            string target = null,
+            int index = 0,
+            CodeActionPriority? priority = null,
+            TestParameters parameters = default)
```

Comment:
do you ever pass priority/index along in tests that call this?  if not, remove.

Reply:
It is provided along with the quick action pull up refactoring. So I use this to by pass these quick action